src/raqm.h

```diff
@@ -156,6 +156,13 @@ RAQM_API raqm_glyph_t *
 raqm_get_glyphs (raqm_t *rq,
                  size_t *length);
 
+RAQM_API raqm_direction_t
+raqm_get_par_resolved_direction (raqm_t *rq);
+
+RAQM_API raqm_direction_t
+raqm_get_direction (raqm_t *rq,
+                    size_t index);
+
```

Comment:
You need to add the new entries to `docs/raqm-sections.txt` and the new API index to `docs/raqm-docs.xml` for the new documentation to be generated (you can catch these issues locally by configuring with `-Ddocs=true` and running `ninja test`.

Reply:
Done